src/main/resources/templates/db_scripts/schema.json

```diff
@@ -536,6 +544,12 @@
           "tOps": "ADD"
         }
       ],
+      "index": [
+        {
+          "fieldName" : "invoiceId",
+          "caseSensitive": false
+        }
+      ],
```

Comment:
Now used when openning invoice from order

Before change:
`15:41:17 [886313/invoice;341263/orders-storage] [diku] [7d5a86d0-f6d7-5826-9b6b-074d323d9437] [mod_orders_storage] WARN  CQL2PgJSON           Doing LIKE search without index for order_invoice_relationship.jsonb->>'invoiceId', CQL >>> SQL: invoiceId == bfb90a0e-2c40-4235-a4aa-549149d08065 >>> lower(f_unaccent(order_invoice_relationship.jsonb->>'invoiceId')) LIKE lower(f_unaccent('bfb90a0e-2c40-4235-a4aa-549149d08065'))
`